content/en/docs/faas/lambda-collector.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Lambda Collector Configuration
+linkTitle: Lambda Collector Configuration
+weight: 11
+description: Add and configure the Collector Lambda layer to your Lambda
+---
+
+The OpenTelemetry community offers the Collector in a separate Lambda layer from
+the instrumentation layers to give users maximum flexibility. This is different
+than the current AWS Distribution of OpenTelemetry (ADOT) implementation which
+bundles instrumentation and the Collector together.
+
+### Add the ARN of the OTel Collector Lambda layer
+
+Once you've instrumented your application you should add the Collector Lambda
+layer to collect and submit your data to your chosen backend.
+
+Note: Lambda layers are a regionalized resource, meaning that they can only be
+used in the Region in which they are published. Make sure to use the layer in
+the same region as your Lambda functions.
+
+Find the supported regions and amd64(x86_64)/arm64 layer ARN in the table below
+for the ARNs to consume.
+
+### Configure the OTel Collector
+
+The configuration of the OTel Collector Lambda layer follows the OpenTelemetry
+standard.
+
+By default, the OTel Collector Lambda layer uses the config.yaml.
+
+#### Set the Environment Variable for your Preferred Backend
+
+In the Lambda environment variable settings create a new variable that holds
+your authorization token.
+
+#### Update the Default Exporters
+
+In your `config.yaml` file add your preferred exporter(s) if they are not
+already present. Configure your exporter(s) using the environment variables you
+set for your access tokens in the previous step.
+
+Here is a sample configuration file of collector.yaml in the root directory:
+
+```yaml
+#collector.yaml in the root directory
+#Set an environemnt variable 'OPENTELEMETRY_COLLECTOR_CONFIG_FILE' to '/var/task/collector.yaml'
+
```

Comment:
```suggestion
```

I would move the example setting of the env. var. to the paragraph below.

Reply:
@tylerbenson would the environment variable be our default recommendation or an advanced config?

Reply:
Without an environment variable being set, the default configuration only supports emitting data using the logging exporter